monkestation/code/modules/new_antagonists/slasher/abilities/soul_steal.dm

```diff
@@ -0,0 +1,68 @@
+/mob/living/carbon/human
+	/// Has our soul been sucked, this makes us pale white.
+	var/soul_sucked = FALSE
+	///sucked precent
+	var/sucked_precent = 0
```

Comment:
its going to be so funny when someone finds this var in the code without context